bash/hail

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I didn't want to put them in a `bin` directory of the repository because they are not executable in a freshly downloaded repository, even after building the JAR (in particular: the JAR won't be in the correct location). They're just some scripts that make sense in the context of the distribution.

If I understand you correctly, we're using the second approach. All the necessary files are copied into `build/package` and then that folder is zipped.